docs/installing.rst

```diff
@@ -148,8 +148,22 @@ The following virtualization providers are suggested for the respective operatin
     # Set libvirtd to start automatically on system boot
     $ sudo systemctl enable libvirtd
 
+---------------------------------------------------
+3. Install dependencies for vagrant-service-manager
+---------------------------------------------------
+
+For some operating systems, you might need to install additional dependencies.
+
+* Fedora 24
+
+  Run the following commands to install ``Development Tools``, ``rpm-build``, and ``zlib-devel`` packages::
+
+  $ sudo dnf install "Development Tools"
+  $ sudo dnf install rpm-build
+  $ sudo dnf install zlib-devel
```

Comment:
I believe you are missing ruby-devel:

     sudo dnf install -y vagrant vagrant-libvirt
     sudo dnf install -y @"Development Tools" rpm-build gcc-c++
     sudo dnf install -y ruby-devel zlib-devel

Who is verifying the required commands set?